velox/expression/tests/MapViewTest.cpp

```diff
@@ -514,4 +502,14 @@ TEST_F(NullFreeMapViewTest, testReadingStructureBindingLoop) {
   readingStructureBindingLoopTest();
 }
 
+TEST_F(NullFreeMapViewTest, testSubscript) {
+  auto mapVector = createTestMapVector();
+  DecodedVector decoded;
+  exec::VectorReader<Map<int64_t, int64_t>> reader(
+      decode(decoded, *mapVector.get()));
+
+  ASSERT_THROW(read(reader, 1)[5], VeloxException);
+  ASSERT_EQ(read(reader, 1)[3], 3);
```

Comment:
nit: it'd be a little better to test the entry with key 1 value 4, here since the key and value are the same this test could miss it if you swapped the key and value vectors inside the view